pom.xml

```diff
@@ -82,6 +82,19 @@
         <module>integration-tests</module>
     </modules>
 
+    <repositories>
+        <repository>
+            <id>jboss-public-repository-group</id>
+            <name>JBoss Public Repository</name>
+            <snapshots>
+              <enabled>false</enabled>
```

Comment:
Maybe you should enable the snapshots if you want them to work :].

Reply:
😅😅😅😅